ec2_metadata.py

```diff
@@ -83,5 +83,10 @@ def region(self):
     def reservation_id(self):
         return requests.get(METADATA_URL + 'reservation-id').text
 
+    @cached_property
+    def security_groups(self):
+        # since this can be one or more items, always return as an array.
```

Comment:
this comment is pretty vacuous considering the name of the property is `security_groups`, please remove 👌